src/html/partials/story/intro.hbs

```diff
@@ -186,7 +184,7 @@ By 1983, emotional ballads like “I Will Survive,” with their roots in Soul m
 </p>
 <!-- Crowd noises? Grimes ref. https://soundcloud.com/hrishihirway/song-exploder-grimes -->
 
-<p class="step" data-song="Livin' On A Prayer">
+<p class="step" data-song="Livin’ on a Prayer">
```

Comment:
I think you failed to update this title in the csv. The js currently fails to find the song when we reach here. 

(FWIW, I think it might have been easier to write some js code to translate the original names to their display versions (by converting apostrophes and downcasing particles) rather than changing the csv. But this way seems fine too.)

Reply:
I thought about that, but was hesitant to ask, and a few of them wouldn't have come out right anyways (at least one had the artist as "last name, first name"), so I figured if I was cleaning the CSV, I may as well clean the CSV. I'll definitely do another pass on it before we publish